src/diagnostic-channel-publishers/package.json

```diff
@@ -4,16 +4,15 @@
   "main": ".dist/src/index.js",
   "types": ".dist/src/index.d.ts",
   "scripts": {
-    "build": "tsc --outDir ./.dist && node copyTestAssets.js",
-    "lint": "tslint -c tslint.json -p tsconfig.json",
+    "build": "node node_modules/typescript/bin/tsc --outDir ./.dist && node copyTestAssets.js",
```

Comment:
Are you sure that specifying these full paths are required? I thought that when running npm scripts it added the `node_modules/.bin` folder to the PATH to it should find tsc and tslint since they are dependencies